plugins/search-backend-module-pg/src/database/util.test.ts

```diff
@@ -38,7 +38,7 @@ describe('util', () => {
 
   describe('supported', () => {
     const databases = TestDatabases.create({
-      ids: ['POSTGRES_13', 'POSTGRES_9'],
+      ids: ['POSTGRES_16', 'POSTGRES_12'],
```

Comment:
now that its changed to adding 12 and 16 instead of replacing the old ones, what do you think about keeping the test as it was but add additional ids to to test the additional supported versions as well?

Reply:
The goal of these test changes is to align them with the Postgres Policy we want to commit to which is to support and test towards the last supported version and the latest version. I'm following the guidance that @Rugvip suggested at the end of this comment: https://github.com/backstage/backstage/issues/21234#issuecomment-1807967207. I think just continuing to add will impact the overall performance and time it takes to run the test suite. Does that make sense?

Reply:
Indeed. Also, in CI we'll actually cut the list down to only the ones actually enabled by the presence of env variables pointing to instances.

https://github.com/backstage/backstage/blob/master/.github/workflows/ci.yml#L156

https://github.com/backstage/backstage/blob/master/.github/workflows/ci.yml#L221

If some plugins explicitly want or need to deviate from the policy then we can consider it but we'll have to enable more databases in CI then.